NewHorizons/Patches/RaftPatches.cs

```diff
@@ -104,5 +104,14 @@ public static bool AlignToSurfaceFluidDetector_ManagedFixedUpdate(AlignToSurface
 
             return false;
         }
+
+        [HarmonyPrefix]
+        [HarmonyPatch(typeof(FluidVolume), "GetDepthAtPosition")]
+        public static bool SphereOceanFluidVolume_GetDepthAtPosition(SphereOceanFluidVolume __instance, ref float __result, Vector3 worldPosition)
```

Comment:
This is neat, you can patch an inherited method that isn’t overridden? I didn’t know that. Nice

Reply:
afaik you cant????????? i would think this would break if FluidVolume isnt a SphereOceanFluidVolume

Reply:
Ok. I'll change it then.